api/routes/settings.ts

```diff
@@ -0,0 +1,70 @@
+import { FastifyPluginCallback } from 'fastify';
```

Comment:
```suggestion
import type { FastifyPluginCallbackTypebox } from '@fastify/type-provider-typebox';
```
Niraj spotted this one: it's a shorthand that gets us type safety automatically.